test/fuzz/test_runner.py

```diff
@@ -152,6 +167,35 @@ def main():
         )
 
 
+def generate_corpus_seeds(fuzz_pool, build_dir, seed_dir, targets, n):
+    """Generates new corpus seeds.
+
+    Run {n} {targets} at a time without input, and outputs the generated corpus
+    seeds to {seed_dir}.
+    """
+    logging.info(f"Generating corpus seeds to {seed_dir}")
```

Comment:
Extraneous `f` in `f"Generating..."`

Reply:
That's an [fstring](https://www.python.org/dev/peps/pep-0498/).

Reply:
Had to drop it anyway because we need to support python 3.5 !